controllers/compassmanager_controller_test.go

```diff
@@ -169,7 +199,7 @@ func createKymaResource(name string) kyma.Kyma {
 	}
 }
 
-func createCredentialsSecret(kymaName, namespace string) corev1.Secret {
+func createCredentialsSecret(kymaName, namespace string) corev1.Secret { //nolint:unparam
```

Comment:
Why do we silence the `unparam`? If we're always working in the same namespace, maybe it isn't needed.

Reply:
`getCompassMappingCompassID` has no namespace param